tests/csm/rest/test_capacity_quota.py

```diff
@@ -0,0 +1,442 @@
+        resp1 = self.csm_obj.compare_iam_payload_response(resp, payload)
+        self.log.info("Printing response %s", resp1)
+        assert_utils.assert_true(resp1[0], resp1[1])
+        self.log.info("Step 2: Create bucket under above IAM user")
+        self.akey = resp.json()["access_key"]
+        self.skey = resp.json()["secret_key"]
+        self.bucket_name = "iam-user-bucket-" + str(int(time.time()))
+        self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
+        self.bucket_name, self.akey, self.skey)
+        bucket_created = s3_misc.create_bucket(self.bucket_name, self.akey, self.skey)
+        assert bucket_created, "Failed to create bucket"
+        self.log.info("Step 3: Perform PUT API to set user level quota with random values")
+        uid = resp1.json()['tenant'] + "$" + payload["uid"]
+        test_cfg = self.csm_conf["test_40632"]
+        max_size = test_cfg["max_size"]
+        max_objects = test_cfg["max_objects"]
+        resp3 = self.quota_obj.set_user_quota(uid, "user","true", max_size, max_objects)
+        assert_utils.assert_true(resp3[0],resp3[1])
+        self.log.info("Step 4: Perform GET API to get user level quota")
+        resp4 = self.quota_obj.get_user_quota(uid, "user")
+        assert_utils.assert_true(resp4[0], resp4[1])
+        self.log.info("Step 5: Perform Put operation for 1 object of max size")
+        resp = s3_misc.create_put_objects(self.obj_name, self.bucket_name,
+                                          self.akey, self.skey, object_size=max_size)
+        assert_utils.assert_true(resp, "Put object Failed")
+        self.log.info("Step 6: Perform Put operation of Random size and 1 object")
+        random_size = self.cryptogen.randrange(1, max_size)
+        resp = s3_misc.create_put_objects(self.obj_name, self.bucket_name,
+                                          self.akey, self.skey, object_size=random_size)
+        assert_utils.assert_true(resp, "Put object Failed")
+        self.log.info("Step 7: Delete object")
+        assert s3_misc.delete_object(
+            self.bucket, self.obj_name, self.akey, self.skey), "Failed to delete bucket."
+        self.log.info("Step 8: Perform Put operation of small size and N object")
+        small_size = math.floor(max_size/max_objects)
+        for _ in range(0, max_objects):
+            resp = s3_misc.create_put_objects(self.obj_name, self.bucket_name,
+                                              self.akey, self.skey, object_size=small_size)
+            assert_utils.assert_true(resp, "Put object Failed")
+        self.log.info("Step 9: Perform Put operation of Random size and 1 object")
+        resp = s3_misc.create_put_objects(self.obj_name, self.bucket_name,
+                                          self.akey, self.skey, object_size=random_size)
+        assert_utils.assert_true(resp, "Put object Failed")
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
```

Comment:
Above comments are applicable for all the further tests.

Reply:
okay